cristian-bars/challenge/precurse/Calculadora_HTML/script/script.js

```diff
@@ -0,0 +1,73 @@
+let operandoa = "";
+let operandob = "";
+let operacion = "";
+let flag = false;
+
+function add(key) { 
+    if(operandoa !== "" && flag !== true){
+        result.textContent = "";
+        flag = true;
+    }
+    result.textContent = result.textContent + key;
+}
+
+function del(){
```

Comment:
It is better and more readable to define full word functions. For example, 'delete' instead of 'del' or 'arrow' instead of 'arr'.

Reply:
Done